Vagrantfile

```diff
@@ -22,7 +22,6 @@ Vagrant.configure("2") do |config|
     apt install -y python3-pip
     # install X11 for matplotlib
     apt install -y xserver-xorg-core x11-utils x11-apps
-
```

Comment:
Why are these removed? It was easier to read. Are the blank spaces causing any issues with running Vagrant?

Reply:
I'm not sure what happened, I committed them back in.

Reply:
Thanks!